src/bsp/src/stm32/src/USB.cpp

```diff
@@ -0,0 +1,46 @@
+#include "hal/usb.h"
+#include "usbd_cdc_if.h"
+#include <USB.h>
+
+bool USB::send(const uint8_t* buffer, uint16_t length) {
+    if (buffer == nullptr || length > CBUFF_USB_DATA_SIZE) {
+        m_logger.log(LogLevel::Warn, "Invalid parameters for USB::send");
+        return false;
+    }
+
+    bool ret = false;
+    int out = Usb_Send_Data(const_cast<uint8_t*>(buffer), length);
+
+    if(out == USBD_OK) {
```

Comment:
Don't forget to change for our custom enum

Reply:
done